### ISSUE TITLE
[priest] Psychic Scream/other fear mb

### ISSUE THREAD

**Issue by mipanka** [gamefreedomgit/Maelstrom#5684]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**
player with effect https://wowpedia.fandom.com/wiki/Psychic_Scream is visually in place, but actually moves to another place at the end of the end effect https://wowpedia.fandom.com/wiki/Psychic_Scream

**How to reproduce:**
need 3 players one will fear enemy and ally shoul try attack him or use melee attack

**How it should work:**
player in https://wowpedia.fandom.com/wiki/Psychic_Scream or other simillar effect should move without "blink"

**Database links:**
https://wowpedia.fandom.com/wiki/Psychic_Scream 
and simillar ability (not tested) 
https://wowpedia.fandom.com/wiki/Fear
https://wowpedia.fandom.com/wiki/Howl_of_Terror

**Comment by UltraNix** [comment]
@mipanka @Pornoodemon As I can't reproduce it - can you record it?
Also - with or without glyph?

**Comment by mipanka** [comment]
Without glyph. Ok, I will record it

**Comment by SimonBpvp** [comment]
Yeah fear is bugged right now. It doesn’t track where the feared player actually is on your screen so your spells can be all out of range especially for melees. Apollo had same bug for a while I remember.

**Comment by SimonBpvp** [comment]
Still bugged atm

**Comment by UltraNix** [comment]
Still waiting for that video...

**Comment by Pornoodemon** [comment]
> Все еще жду это видео...

I can't figure out how this bug works
This happens sometimes, but I can't reproduce it myself either.

**Comment by SimonBpvp** [comment]
https://www.twitch.tv/videos/2095686997?t=0h44m11s

this is one incidence of it happening - albiet a quite benign example of it

**Comment by UltraNix** [comment]
@SimonBpvp Video not present.

**Comment by SimonBpvp** [comment]
my vods only last 1 week cause i'm a pleb

**Comment by SimonBpvp** [comment]
https://clips.twitch.tv/UninterestedOptimisticRamenTF2John-p7yOhV6OHex2arFu

Here's a new clip and it doesn't expire after one week (but don't wait a week to look at it though pls)

**Comment by SimonBpvp** [comment]
bump
